edx/frontend-app-account
# Modularizing DeleteAccount

Puts all the code related to account deletion into a "delete-account" submodule of "account-settings".  reducers, actions, sagas, and services are put into a "data" sub-directory.

# Testing

- Ensure Delete My Account section still appears
- Ensure printing instructions appear in section and in confirmation modal
- Ensure putting in a bad password returns a server error/highlights the field
- Ensure deleting the account actually works 
- Ensure success modal appears after delete
- Try refreshing page without closing success modal.  See what happens.
- Ensure redirects to somewhere once success modal is closed
- Ensure activate your account banner appears
- Ensure linked account banner appears
- Ensure cancel button works
- Try to submit without a password
- Unlink an account and see if TPA banner goes away
